release_notes/ocp_3_4_release_notes.adoc

```diff
@@ -119,43 +490,276 @@ Downgrading from Elasticsearch 2.4 to Elasticsearch 1.x is not possible due to
+
+* The `openshift-ansible` NetworkManager configuration script was unconditionally restarting the dnsmasq service every time it ran. As a result, host name resolution would fail temporarily while the dnsmasq service restarted. The `openshift-ansible` NetworkManager configuration script now only restarts the dnsmasq service if a change was detected in the upstream DNS resolvers. As a result, host name resolution will continue to function as expected.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1374170[*BZ#1374170*])
+
+* Previously, the installer would re-run the metrics deployment steps if the configuration playbook was re-run. The playbooks are now updated to only run the metrics deployment tasks once. If a previous installation of metrics has failed, the administrator must manually resolve the issue or remove the metrics deployment and re-run the configuration playbook. See the xref:../install_config/cluster_metrics.adoc#metrics-cleanup[cleanup instructions]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1383901[*BZ#1383901*])
+
+* The Ansible `quiet output` configuration was not set for non-install runs of `atomic-openshift-installer`. As a result, users would see full Ansible output rather than abbreviated step-by-step output. The Ansible `quiet output` configuration is now set as the default for all `atomic-openshift-installer` runs. With this fix, users see abbreviated output and can toggle back to verbose output with `-v `or `--verbose`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1384294[*BZ#1384294*])
+
+* Previously, the quick installer would unnecessarily prompt for the name of a load balancer for non-HA installations. This question has been removed for single master environments.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1388754[*BZ#1388754*])
+
+* The *a-o-i* package was considering extra hosts when determining if the target HA environment is a mix of installed and uninstalled hosts. As a result, the comparison failed and incorrectly reported that a fully installed environment was actually a mix of installed and uninstalled. With this fix, non-masters and non-nodes were removed from the comparison and installed HA environments are correctly detected.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1390064[*BZ#1390064*])
+
+* Previously, the dnsmasq configuration included `strict-order`, meaning that dnsmasq would iterate through the host's nameservers in order. This meant that if the first nameserver had failed, a lengthy timeout would be observed while dnsmasq waited before moving on to the next nameserver. By removing the `strict-order` option, dnsmasq prefers nameservers that it knows to be up over those that are unresponsive, ensuring faster name resolution. If you wish add this or any other option, use the advanced installer option `openshift_node_dnsmasq_additional_config_file`,  which allows you to provide the path to a dnsmasq configuration file that will be deployed on all nodes.
```

Comment:
s/if you wish add this/If you wish to add this

And a double space inbetween "`openshift_node_dnsmasq_additional_config_file`,  which"